holmes.cabal

```diff
@@ -38,14 +38,14 @@ library
                , Data.JoinSemilattice.Class.Zipping
                , Data.CDCL
 
-  build-depends: base >= 4.12 && < 4.16
-               , containers >= 0.6 && < 0.7
-               , hashable >= 1.3 && < 1.4
-               , hedgehog >= 1.0 && < 1.1
-               , logict >= 0.7 && < 0.8
-               , mtl >= 2.2 && < 2.3
-               , primitive >= 0.7 && < 0.8
-               , transformers >= 0.5 && < 0.6
+  build-depends: base >= 4.12 && < 4.19
+               , containers >= 0.6 && < 0.8
+               , hashable >= 1.3 && < 1.5
+               , hedgehog >= 1.0 && < 1.5
+               , logict >= 0.7 && < 0.8.2
```

Comment:
If 0.8.1 has been released and works, PVP says there'll be no breaking changes until 0.9, so you may as well raise the upper bound to that.